suites/squid/rgw/tier-2_rgw_ms_async_data_notification.yaml

```diff
@@ -456,6 +456,20 @@ tests:
       module: sanity_rgw_multisite.py
       name: test_sse_s3_per_bucket_with_notifications_dynamic_reshard
       polarion-id: CEPH-83586489
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            run-on-rgw: true
+            extra-pkgs:
+              - wget https://download.oracle.com/java/23/latest/jdk-23_linux-x64_bin.rpm
+            install_start_kafka: true
```

Comment:
@viduship 
FYI,
install_start_kafka is redundant as the previous test holds it..
this will not harm even if it present I think

Reply:
updated @hmaheswa

Reply:
done @hmaheswa